### PR TITLE
add documentation for LDAP security provider [CN-995]

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

**Review by semihbkgr — APPROVED** [review]

**Review by cagric0 — APPROVED** [review]

File: docs/modules/ROOT/examples/management-center-ldap.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: ManagementCenter
+metadata:
+  name: managementcenter
+spec:
+  repository: "hazelcast/management-center"
+  licenseKeySecretName: hazelcast-license-key
+  securityProvider:
+    ldap:
+      credentialsSecretName: ldap-credentianls
```
[the remark this answers is not in the document]
Comment (SeriyBg) [reply docs/modules/ROOT/examples/management-center-ldap.yaml:10]: Done

File: docs/modules/ROOT/pages/management-center-ldap.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+To set up LDAP, you need to configure the `securityProvider.ldap` section in the Management Center CR.
+The following are the required field to configure the LDAP Security provider.
+
+[cols="30%m,70%a"]
+|===
+|Field|Description
+
+|`credentialsSecretName`
+| The name of the secret that contains `username` and `password` keys of a user that has admin privileges on the LDAP server. The `username` must be the DN of the user. It is used to connect to the server when authenticating users.
+
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/management-center-ldap.adoc:19]: Maybe it is better to show the expanded word for DN when it is used for the first time. Something like:

"... must be the distinguished name (DN) of the user. ..."

File: docs/modules/ROOT/pages/management-center-ldap.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+|`userSearchFilter`
+| LDAP search filter expression to search for the users.
+
+|`groupSearchFilter`
+| LDAP search filter expression to search for the groups.
+
+|===
+
+=== Example Management Center LDAP configuration
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/management-center-ldap.adoc:46]:
```suggestion
=== Example Management Center LDAP Configuration
```

File: docs/modules/ROOT/pages/management-center-ldap.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+:description: You can use your existing LDAP server for authentication/authorization on the Management Center.
+
+{description}
+
+See xref:management-center:deploy-manage:ldap.adoc[Management Center Documentation] for more detailed information.
+
+== Setting Up the LDAP Security Provider
+
+To set up LDAP, you need to configure the `securityProvider.ldap` section in the Management Center CR.
+The following are the required field to configure the LDAP Security provider.
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/management-center-ldap.adoc:11]:
```suggestion
The following are the required fields to configure the LDAP Security provider.
```

File: docs/modules/ROOT/pages/management-center-ldap.adoc [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+| LDAP search filter expression to search for the users.
+
+|`groupSearchFilter`
+| LDAP search filter expression to search for the groups.
+
+|===
+
+=== Example Management Center LDAP configuration
+
+Bellow is the example configuration for the LDAP Security Provider:
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/management-center-ldap.adoc:48]:
```suggestion
The following is an example configuration for the LDAP Security Provider:
```

File: docs/modules/ROOT/pages/management-center-ldap.adoc [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/management-center-ldap.yaml[]
+----
+
+And the exmaple of how to create a `Secret` for the LDAP credentials:
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/management-center-ldap.adoc:55]:
```suggestion
The following example shows how to create a `Secret` for the LDAP credentials.
```

**Review by hasancelik — APPROVED** [review]

**Review by Serdaro — APPROVED** [review]

**Pull request merged**